src/hoverifier.ts

```diff
@@ -789,10 +789,6 @@ export function createHoverifier<C extends object, D, A>({
                         })
                     }
 
-                    // The requested position is is 0-indexed; the code here is currently 1-indexed
-                    const { line, character } = pos
-                    pos = { line: line + 1, character: character + 1, ...pos }
```

Comment:
**question**: Why was this removed? This is passed back into user-provided functions like `adjustPosition`, so won't this cause problems / is this not a breaking change?

Reply:
The incremented `line` and `character` were overwritten by spreading `pos`. 

Also, it seems that `PositionAdjuster` is meant to receive and return a 0-based position. Furthermore, the position's line and character are incremented by 1 in the next `switchMap` in `getHighlightedRange` anyways. If my assumptions are correct, then there's no need to convert the indexing at this point. If not, then the fix is to spread `pos` before incrementing line and character.

Reply:
Oh haha, you're right.